pyhgvs/utils.py

```diff
@@ -106,7 +106,7 @@ def read_transcripts(refgene_file):
     Read all transcripts in a RefGene file.
     """
     transcripts = {}
-    for trans in imap(make_transcript, read_refgene(refgene_file)):
+    for trans in map(make_transcript, read_refgene(refgene_file)):
```

Comment:
Note that this increases the memory footprint in python 2.X. It might be better to make this a generator comprehension, or just call `make_transcript` inside the `for` loop.